.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
```

Comment:
Can you add a usage guide for these linters in our README? Also because I'm a noob and I still don't know how to use them LOL